felt/src/lib.rs

```diff
@@ -1052,6 +1088,204 @@ mod test {
             let x = Felt::parse_bytes(x.as_bytes(), 10).unwrap();
             let y = Felt::parse_bytes(y.as_bytes(), 10).unwrap();
             prop_assert!(x.is_multiple_of(&y));
+            // prop_assert!(x.divides(&y));
+        }
+
+        #[test]
+        fn divides_doesnt_panic(ref x in "(0|[1-9][0-9]*)", ref y in "(0|[1-9][0-9]*)") {
+            let x = Felt::parse_bytes(x.as_bytes(), 10).unwrap();
+            let y = Felt::parse_bytes(y.as_bytes(), 10).unwrap();
+            prop_assert!(x.divides(&y));
```

Comment:
This test will fail for `y == 0`. I'm not 100% what should happen for `x == 0` if this is not returning a `Result<_>`, but we're passing it in the test as well.

Reply:
It won't fail, `is_multiple_of` and `divides` always return `true`, see [`bigint_felt.rs#L615`](https://github.com/Quadratic-Labs/cairo-rs/blob/ef74e80fe4f8f1156233418e3a2c381db5b14047/felt/src/bigint_felt.rs#L615)

Reply:
That is odd. `0` shouldn't divide anything.